### DISCUSSION TITLE
Overloading logs with INFO level  Header(type:b'MSG', chunk_type:b'F', body_size:62, channel:2612142770)

### DISCUSSION THREAD

**Discussion by pavlo-yashchuk** [FreeOpcUa/python-opcua#1429] [Q&A] [answered]
I create opc client which connects to differents server and after connection if logs turn on I has too much INFO logs like this 
![image](https://user-images.githubusercontent.com/85494277/146364676-d8de4fe4-a484-488c-b00b-0e18f5fb2a2e.png)
and these messages when stopped client
![image](https://user-images.githubusercontent.com/85494277/146364806-adc8f8d8-ea95-4998-bcfc-d18faa0a15d9.png)
What is it? And how can it be solved?

**Comment by AndreasHeine** [comment]
what logginglevel do you use?

**Reply by pavlo-yashchuk** [reply]
I use level = INFO
Is to cover client with logs

**Reply by swamper123** [reply]
Do you need the INFO level?
Would WARNING be enough?

EDIT: The logs you got are normal logs on INFO-level of our server. You find them inside the clases (e.g. opcua -> client -> ua_client) in some \_\_init\_\_  methods.

**Reply by pavlo-yashchuk** [reply]
But its not solve that) 
Or that its normal?

**Reply by pavlo-yashchuk** [reply]
Hmm

**Reply by pavlo-yashchuk** [reply]
I see, then I have to give up information logs
:(

**Reply by pavlo-yashchuk** [reply]
Thanks!

**Reply by swamper123** [reply]
Well, just because some stuff was logged, it doesn't mean it is a problem or sth. wasn't working (in that case, it would be a log on WARNING or ERROR level). But it is normal, that the INFO level has ... many infos. On DEBUG level you even got more of them.

If you want to know, what happens on INFO-Level on the server, you have to live with it (or fork it and make your own, logger-less version of python-opcua). This is what loggers are made for. 

I am not familiar with handling multiple loggers in python (so far), but I bet there are posibilities to turn them off (on own risk of course) or set their levels.

**Comment by pavlo-yashchuk** [comment]
Thanks!

**Comment by oroulet** [comment, marked as answer]
```
ualogger = logging.getLoger("asyncua")
ualogger.setLevel(logging.CRITICAL)
```

That would make only the ua part of logger silent

**Reply by pavlo-yashchuk** [reply]
Thanks)
